examples/tokenization/tokenization_example.py

```diff
@@ -0,0 +1,136 @@
+# This is just for very basic implementation reference, in production, you should validate the incoming requests and implement your backend more securely.
+import midtransclient
+import datetime
+
+# Initialize core api client object
+# You can find it in Merchant Portal -> Settings -> Access keys
+USED_SERVER_KEY = 'YOUR_SERVER_KEY'
+USED_CLIENT_KEY = 'YOUR_CLIENT_KEY'
+
+tokenization = midtransclient.Tokenization(
+    is_production=False,
+    server_key=USED_SERVER_KEY,
+    client_key=USED_CLIENT_KEY
+)
+
+core_api = midtransclient.CoreApi(
+    is_production=False,
+    server_key=USED_SERVER_KEY,
+    client_key=USED_CLIENT_KEY
+)
+
+# prepare parameter ( refer to: https://api-docs.midtrans.com/#create-pay-account )
+param = {
+  "payment_type": "gopay",
+  "gopay_partner": {
+    "phone_number": "81234567891",
+    "country_code": "62",
+    "redirect_url": "https://midtrans.com"
+  }
+}
+
+# link payment account
+link_payment_account = tokenization.link_account(param)
+print('link_payment_account_response:')
+print(link_payment_account)
+
+# link_payment_account_response is dictionary representation of API JSON response
+# sample:
+# {
+#     "status_code": "201",
+#     "payment_type": "gopay",
+#     "account_id": "6e902848-c4c3-4f40-abe6-109e8749df21",
+#     "account_status": "PENDING",
+#     "actions": [
+#         {
+#             "name": "activation-deeplink",
+#             "method": "GET",
+#             "url": "https://api-v2.sandbox.midtrans.com/v2/pay/account/gpar_2b78cfea-2afa-49b3-86d5-3866626ce015/link"
+#         },
+#         {
+#             "name": "activation-link-url",
+#             "method": "GET",
+#             "url": "https://api-v2.sandbox.midtrans.com/v2/pay/account/gpar_2b78cfea-2afa-49b3-86d5-3866626ce015/link"
+#         },
+#         {
+#             "name": "activation-link-app",
+#             "method": "GET",
+#             "url": "https://simulator-v2.sandbox.midtrans.com/gopay/partner/web/otp?id=e4514b08-cc16-486e-9e4a-d8d8dd0bfe49"
+#         }
+#     ],
+#     "metadata": {
+#         "reference_id": "48b854c9-cc65-4af2-a3f0-38ae81798512"
+#     }
+# }
+# for the first link, the account status is PENDING, you must activate it by accessing one of the URLs on the actions object
+
+active_account_id = "6975fc98-8d44-490d-b50a-28d2810d6856"
```

Comment:
If this is hardcoded sample account_id, please add comment that says this is a hardcoded sample account id that has been already activated before